File: PB - More Exercises Python/7.0 Nested Loops - More Exercises/09. Sum of Two Numbers.py
start, end, magic_num = [int(input()) for _ in range(3)]
counter = 0

for a in range(start, end + 1 ):
    for b in range(start, end + 1):
        counter += 1
        if a + b == magic_num:
            print(f'Combination N:{counter} ({a} + {b} = {a + b})')
            exit()

print(f'{counter} combinations - neither equals {magic_num}')


'''
23
24
20
'''

File: PB - More Exercises GO/7.0 Nested Loops - More Exercises/09. Sum of Two Numbers.go
package main

import (
	"fmt"
	"os"
)

func main() {
	var start, end, magicNum, counter int
	fmt.Scan(&start, &end, &magicNum)

	for a := start; a <= end; a++ {
		for b := start; b <= end; b++ {
			counter++
			if a+b == magicNum {
				fmt.Printf("Combination N:%v (%v + %v = %v)\n", counter, a, b, a+b)
				os.Exit(0)
			}
		}
	}

	fmt.Printf("%v combinations - neither equals %v\n", counter, magicNum)

}

/*
name   :09. Sum of Two Numbers
input  :1 10 5
output :Combination N:4 (1 + 4 = 5)
*/
